### PR TITLE
CouchbaseContainer fixes

### PR DESCRIPTION
**Pull request by kaidowei**
- removed example buckets
- added missing withClusterUsername method
- fixed capi port mapping:
  the environment patching was removed from the couchbase docker container.
  couchbase/docker@86e665b#diff-8941c93c7af57eff48058be15eb5ecff
- updated container from 5.1.0 to 5.1.1

### CODE REVIEW

**Comment by kaidowei** [comment]
will rebase when https://github.com/testcontainers/testcontainers-java/pull/829 is merged.

File: modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java [hunk 1]
```diff
@@ -67,12 +61,11 @@
 @AllArgsConstructor
 public class CouchbaseContainer extends GenericContainer<CouchbaseContainer> {
 
-    public static final String VERSION = "5.1.0";
+    public static final String VERSION = "5.1.1";
```
Comment (daschl) [inline modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java]: any reason we stick to 5.1.1 here? 5.5.0 is out. Is there a reason it is not using latest if not otherwise specified explicitly by the user?

File: modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java [hunk 2]
```diff
@@ -392,6 +358,11 @@ public CouchbaseContainer withIndexMemoryQuota(String indexMemoryQuota) {
         return self();
     }
 
+    public CouchbaseContainer withClusterUsername(String clusterUsername) {
```
Comment (daschl) [inline modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java]: I wonder if this should be withAdminUsername / withAdminPassword ? to distinguish it from real RBAC users

File: core/src/main/java/org/testcontainers/containers/VncRecordingSidekickContainer.java [hunk 3]
```diff
@@ -1,94 +0,0 @@
-package org.testcontainers.containers;
```
Comment (daschl) [inline core/src/main/java/org/testcontainers/containers/VncRecordingSidekickContainer.java]: is this an unrelated change?

File: modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java (hunk 1 above)
Comment (kaidowei) [reply modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java]: api compatibility. But this release will have breaking changes anyway for couchbase containers, so why not

File: core/src/main/java/org/testcontainers/containers/VncRecordingSidekickContainer.java (hunk 3 above)
Comment (kaidowei) [reply core/src/main/java/org/testcontainers/containers/VncRecordingSidekickContainer.java]: there are a lot of changes coming from @bsideup's pr

File: modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java (hunk 2 above)
Comment (kaidowei) [reply modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java]: I like the word cluster for this context. it allows me to manage the whole thing.

Comment (daschl) [reply modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java]: what about ClusterAdmin ? makes both the scope and the role clear.

File: modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java (hunk 1 above)
Comment (daschl) [reply modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java]: 👍

**Comment by rnorth** [comment]
We have this out in a Release Candidate build (1.9.0-rc1) for anyone who is keen to try it!

[Release notes](https://github.com/testcontainers/testcontainers-java/releases/tag/1.9.0-rc1)
